app/questionnaire/rules.py

```diff
@@ -138,17 +138,7 @@ def get_date_match_value(date_comparison, answer_store, schema, metadata):
 
 
 def convert_to_datetime(value: Optional[str]) -> Optional[datetime]:
-    if not value:
-        return None
-
-    if re.match(r"\d{4}-\d{2}-\d{2}", value):
-        date_format = "%Y-%m-%d"
-    elif re.match(r"\d{4}$", value):
-        date_format = "%Y"
-    else:
-        date_format = "%Y-%m"
-
-    return datetime.strptime(value, date_format).replace(tzinfo=timezone.utc)
+    return parse_datetime(value)
```

Comment:
Didn't update calls to `parse_datetime` to keep changes to a minimum, but happy to remove this function.

Reply:
Do you mean calls to `convert_to_datetime`? I think if it's a simple refactor it may add noise but it's easy to see what's happened

Reply:
Ya, it was just to reduce noise, the refactor **should** be a drop-in replacement. 🤞

Reply:
@LJBabbage @petechd Since you already ticked and this was left here beforehand, happy for me to change?

Reply:
@MebinAbraham yeah happy with that!

Reply:
Yep

Reply:
Done in [`d1f2892` (#725)](https://github.com/ONSdigital/eq-questionnaire-runner/pull/725/commits/d1f28929e0cdf1cbdc52f334475871b288adfcde)
@jamesnunn @petechd @LJBabbage